src/VS.Web.CG.Mvc/VS.Web.CG.Mvc.csproj

```diff
@@ -42,9 +44,18 @@
     <None Pack="true" Include="Templates\Identity\wwwroot\lib\jquery-validation\dist\*" PackagePath="Templates\Identity\wwwroot\lib\jquery-validation\dist\" />
     <None Pack="true" Include="Templates\Identity\wwwroot\lib\jquery-validation-unobtrusive\*" PackagePath="Templates\Identity\wwwroot\lib\jquery-validation-unobtrusive\" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Remove="Templates\Identity\SupportPages\**" />
```

Comment:
Might be good to exclude the entire Templates folder here.

Reply:
I'm also going to get rid of the `EmbeddedResourceRemove` and `None Remove` here, it's just VS cruft.